utils/get_aliases.sh

```diff
@@ -90,12 +90,6 @@ function list_pages {
 		echo "${man_child[@]}" >> $map_file
 	fi
 
-	if [ "$parent" == "librpmem" ]; then
```

Comment:
It seems to me that the problem is due to the fact that we removed files from the /doc documentation folder, and on their basis was created an alias map. Now when the map is incomplete documentation for the old versions is not displayed. This is my first thought on this issue.

Reply:
Please undo this change until the fix is figured out, without this change I will be able to merge this PR.

Reply:
Done